Source/DafnyCore/Rewriters/ExpectContracts.cs

```diff
@@ -222,31 +221,36 @@ public class ExpectContracts : IRewriter {
   }
 
   /// <summary>
-  /// Add wrappers for certain top-level declarations in the given module.
-  /// This runs after the first pass of resolution so that it has access to
-  /// ghostness information, attributes and call targets.
+  /// Add wrappers for certain top-level declarations in the given program.
```

Comment:
```suggestion
  /// Adds wrappers for certain top-level declarations in the given program.
```
Usually third person to talk about what a function does